### ISSUE TITLE
--zls in any location

### ISSUE THREAD

**Issue by Jaxon-teBoekhorst** [tristanisham/zvm#107]
When installing a new version of Zig eg: master the command to install with ZLS is zvm i --zls master, yet sometimes i forget that to add the zls tag until after typing the version, it would be convinent to be able to type zvm i master --zls, which is currently not possible

**Comment by tristanisham** [comment]
Worth considering! It definitely annoys me too. Just never enough to get around to changing it. Let me take a look.

**Comment by Jaxon-teBoekhorst** [comment]
looking at urfave/cli, flags in any order is only possible in v3, which the most updated version is v3.0.0-beta1, I'm not sure if you want to use beta dependencies in your project, but if so I think I have a working version with the updated cli

**Comment by tristanisham** [comment]
Yeah, as of now until I get the time to actually dive into it, I don't think I'll be updating to use the beta. If you want to make a PR I'll take a look. I'm not against using beta software, especially in GO. I'd just need to test to make sure nothing broke.

**Comment by tristanisham** [comment]
Live in v0.8.5
